starter-code/src/clue.js

```diff
@@ -1,94 +1,160 @@
+    age:          62,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+    occupation:   "Retired Football player"
+};
 
 // Weapons
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+var rope = {name: "rope",  weight: 10};
+var knife = {name: "knife",  weight: 8};
+var candlestick = {name: "candlestick",  weight: 2};
+var dumbbell = {name: "dumbbell",  weight: 30};
+var poison = {name: "poison",  weight: 2};
+var axe = {name: "axe",  weight: 15};
+var bat = {name: "bat",  weight: 13};
+var trophy = {name: "trophy",  weight: 25};
+var pistol = {name: "pistol",  weight: 20};
+
 
 // Rooms
-name: Dinning Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+var dinningRoom = {name: "Dinning Room"};
+var conservatory = {name: "Conservatory"};
+var kitchen = {name: "Kitchen"};
+var study = {name: "Study"};
+var library = {name: "Library"};
+var billiardRoom = {name: "Billiard Room"};
+var lounge = {name: "Lounge"};
+var ballroom = {name: "Ballroom"};
+var hall = {name: "Hall"};
+var spa = {name: "Spa"};
+var livingRoom = {name: "Living Room"};
+var observatory = {name: "Observatory"};
+var theater = {name: "Theater"};
+var guestHouse = {name: "Guest House"};
+var patio = {name: "Patio"};
+
 
 // Characters Collection
 var charactersArray = [];
+charactersArray.push(mrGreen)
+charactersArray.push(profPlum)
+charactersArray.push(drOrchid)
+charactersArray.push(missScarlet)
+charactersArray.push(mrsPeacock)
+charactersArray.push(mrMustard)
+
 
 // Rooms' Collection
 var roomsArray = [];
+roomsArray.push(dinningRoom);
+roomsArray.push(conservatory);
+roomsArray.push(kitchen);
+roomsArray.push(study);
+roomsArray.push(library);
+roomsArray.push(billiardRoom);
+roomsArray.push(lounge);
+roomsArray.push(ballroom);
+roomsArray.push(hall);
+roomsArray.push(spa);
+roomsArray.push(livingRoom);
+roomsArray.push(observatory);
+roomsArray.push(theater);
+roomsArray.push(guestHouse);
+roomsArray.push(patio);
 
 // Weapons Collection
 var weaponsArray = [];
+weaponsArray.push(rope);
+weaponsArray.push(knife);
+weaponsArray.push(candlestick);
+weaponsArray.push(dumbbell);
+weaponsArray.push(poison);
+weaponsArray.push(axe);
+weaponsArray.push(bat);
+weaponsArray.push(trophy);
+weaponsArray.push(pistol);
+
+
+// -- the code
+
+
+
+
+function randomSelector(array) {
+    var item = array[Math.floor(Math.random() * array.length)];
+    return item;
+};
+
+
+var killer = randomSelector(charactersArray);
```

Comment:
Better do this three selections inside the pickMistery function, it is the only one wich is going to use them.